src/scripts/LuaEngine/UnitFunctions.h

```diff
@@ -2035,15 +2035,23 @@ class LuaUnit
 
     static int ReturnToSpawnPoint(lua_State* L, Unit* ptr)
     {
-        if (ptr == nullptr)
+
+		if (ptr == nullptr)
+        {
             return 0;
+        }
 
         float x = ptr->GetSpawnX();
         float y = ptr->GetSpawnY();
         float z = ptr->GetSpawnZ();
         float o = ptr->GetSpawnO();
-        if (ptr->IsCreature())
-            ptr->GetAIInterface()->MoveTo(x, y, z, o);
+
+		if (ptr->IsCreature())
```

Comment:
Goddamn, don't commit before your fist coffee... Will recommit when I am home and got my hands on my sourcefiles. :-)